test/e2e/workflow/deployment_test.go

```diff
@@ -198,3 +232,51 @@ func ignoreInitialKubeMacPoolRestart() {
 	By("Ignoring initial KubeMacPool restart")
 	time.Sleep(10 * time.Second)
 }
+
+func retreiveRange(configMap *v1.ConfigMap) (string, string, error) {
```

Comment:
No need to return error here. if the `Eventually` fails it will fail the test also